R/generate_template.R

```diff
@@ -85,7 +90,7 @@
 #' create_new_application(name = 'mytestapp', location = tempdir())
 #'
 #' @export
```

Comment:
the @seealso section goes right above the export - see the logging information for an example

Reply:
still outstanding